### ISSUE TITLE
Resize images shown (not the file itself)

### ISSUE THREAD

**Issue by joelteixeira** [giuspen/cherrytree#336]
Hey there, just found out about CT. Congratulations, seems really neat and has tons of features I don't have natively as a ZimWiki user.  I'm using both right now, testing and checking the features and one thing I, personally, would like to change is the way CT handles the images. 

If I resize the picture (just to look better on the document) it actually resize the image file. Is it possible that CT handles the image like an html img tag? Just changing the size shown preserving the image file?

**Comment by dawonn-haval** [comment]
This is my biggest frustration with CT. 

I want to paste in some rather large screenshots, and be able to shrunk them down normally, but then zoom in on them later if I need to see the details.

**Comment by joelteixeira** [comment]
Would be nice to know if is it a planned feature.

**Comment by giuspen** [comment]
I agree it should be done, for a while I'll be fully on the porting to gtk3 though

**Comment by DiagonalArg** [comment]
@dawonn-haval - I put the zoom request here:
https://github.com/giuspen/cherrytree/issues/746
